Tracking/src/ClicEfficiencyCalculator.cc

```diff
@@ -498,11 +500,15 @@ void ClicEfficiencyCalculator::processEvent( LCEvent* evt ) {
           TLorentzVector particleVector2;
           particleVector2.SetPxPyPzE(particle2->getMomentum()[0],particle2->getMomentum()[1],particle2->getMomentum()[2],particle2->getEnergy());
           double DR = particleVector.DeltaR(particleVector2);
+          distancesR.push_back(DR);
           if (DR<0.4) nCloseTrk++;
         }
       }
     }
-    
+    std::vector<double >::iterator result = std::min_element(std::begin(distancesR),std::end(distancesR));
```

Comment:
```c++
double minDR = *result;
```
Should give the resulting DR, i.e., getting the content of the `iterator result`.
NO matter how this is done, it will fail if the vector is empty.